src/main/resources/Diagram/DiagramViewSheet.xml

```diff
@@ -40,7 +40,7 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{velocity}}
-#if ($doc.getObject('Diagram.DiagramClass'))
+#if ($doc.getObject('Diagram.DiagramClass') || $request.export == 'url')
```

Comment:
I'm thinking that we may add support for other values in the future (besides "url") so I would simply check that the parameter value is not empty (so that we don't have to update this code each time we add support for a new value).